website/client/js/services/userServices.js

```diff
@@ -244,6 +244,35 @@ angular.module('habitrpg')
           })
         },
 
+        setCron: function (numberOfDays) {
```

Comment:
It's a little weird that these methods are just on the User object (following the pattern we had in place before for addTenGems, etc). When we rewrite the client, these should be isolated so no module other than the debug footer can use it.